pkg/controller/machineautoscaler/machineautoscaler_controller.go

```diff
@@ -94,11 +99,23 @@ func (r *Reconciler) AddToManager(mgr manager.Manager) error {
 				ToRequests: handler.ToRequestsFunc(targetOwnerRequest),
 			})
 
-		if err != nil {
+		// If we get an error indicating that no matching type is registered
+		// with the API, then remove it from the list of supported target GVKs.
+		// If the type is later registered, a restart of the operator will pick
```

Comment:
Does this have to be a forced restart (by a human)?

Reply:
Yes. For now. Seems reasonable to me. The immediate use case for this is just to not crash if one of the upstream or OpenShift groups is missing.